financial/bill_splitter.py

```diff
@@ -0,0 +1,109 @@
+"""
+Program to split bills among a group by given
+- Number of people in group. eg: 5
+- Name of contributer and amount of contribution separated by space. eg: John 100
+
+"""
+
+from typing import List
+
+from typing_extensions import TypedDict
```

Comment:
Why are you importing this from `typing_extensions` instead of `typing`? I believe the code in this repository is targeted 3.8+

Reply:
Okay. I will change